class/defaults.yml

```
@@ -1,6 +1,7 @@
 parameters:
   prometheus_pushgateway:
     namespace: syn-prometheus-pushgateway
+    createNamespace: true
```

consider using `create_namespace` to keep in line with the other parameters (yes I know it's currently a mess across components).

```suggestion
    create_namespace: true
```

Also update the documentation to include this parameter and describe when/why it's necessary.